mantis-control-plane/mantis-control-plane-server/src/main/java/io/mantisrx/master/resourcecluster/ResourceClusterAkkaImpl.java

```diff
@@ -170,4 +172,21 @@ public CompletableFuture<TaskExecutorStatus> getTaskExecutorState(TaskExecutorID
                 .thenApply(TaskExecutorStatus.class::cast)
                 .toCompletableFuture();
     }
+
+    @Override
+    public CompletableFuture<Void> disabledTaskExecutorsAre(Map<String, String> attributes, Instant expiry) {
+        resourceClusterManagerActor.tell(
+            new ResourceClusterActor.DisableTaskExecutorsRequest(attributes, clusterID, expiry), ActorRef.noSender());
+        return CompletableFuture.completedFuture(null);
```

Comment:
why does this need to be a fut? Doesn't seem to actually have any fut to be used.

Reply:
Fixed this. Thanks for pointing it out.